### PR TITLE
[NUI] Support Blend Equaiton

### PR DESCRIPTION
**Pull request by bshsqa**
Signed-off-by: seungho <sbsh.baek@samsung.com>

### CODE REVIEW

**Comment by bshsqa** [comment]
This patch must be merged after the dali-csharp-binder binding patch.
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/247610/

**Comment by bshsqa** [comment]
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-csharp-binder/+/247610/ is merged

File: src/Tizen.NUI/src/internal/Interop/Interop.GraphicsCapabilities.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+{
+    internal static partial class Interop
+    {
+        internal static partial class GraphicsCapabilities
+        {
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_IsBlendEquationSupported")]
+            public static extern bool IsBlendEquationSupported(int blendEquation);
+        }
+    }
+}
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/internal/Interop/Interop.GraphicsCapabilities.cs:15]: please add new line at the end of file from next time~
( https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file )

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Pull request merged**